store/local_cache_supplier_schemes.go

```diff
@@ -18,10 +18,11 @@ func (s *LocalCacheSupplier) handleClusterInvalidateScheme(msg *model.ClusterMes
 }
 
 func (s *LocalCacheSupplier) SchemeSave(ctx context.Context, scheme *model.Scheme, hints ...LayeredStoreHint) *LayeredStoreSupplierResult {
+	result := s.Next().SchemeSave(ctx, scheme, hints...)
 	if len(scheme.Id) != 0 {
-		defer s.doInvalidateCacheCluster(s.schemeCache, scheme.Id)
+		s.doInvalidateCacheCluster(s.schemeCache, scheme.Id)
```

Comment:
As with the `RoleSave()` above, this reordering changes the semantics of what this method does.